cartesian_impedance_msgs/msg/SetCartesianImpedance.msg

```diff
@@ -1,10 +1,13 @@
 # Here we combine all the parameters message definitions into a single message
 # that needs to be send to the controller in order to adjust the Cartesian Impedance Parameters
 
-string               tool
-CartesianStiffness   stiffness
-CartesianDamping     damping
-MaxCartesianVelocity max_cart_vel
-MaxControlForce      max_ctrl_force
-MaxPathDeviation     max_path_deviation
-NullSpace            null_space_params
+string                 tool
+CartesianStiffness     stiffness
+#use this if the damping is in Ns/m, Ns/rad
```

Comment:
I also agree with @gavanderhoorn that we should definitely find consensus whether we should use ratio's or Ns/m, I think that this should highly depend on the units majority of the robots use as well. I am also looking at how to convert from Ns/m to ratios for example and it seems it is not very straight forward thing to do...

Maybe if we can get input from others who have robots who use Cartesian Impedance and the units they use would be great.